provision-cluster/delete.js

```diff
@@ -1,14 +1,30 @@
+'use strict';
 const core = require('@actions/core');
 const github = require('@actions/github');
+const kubeception = require('./lib/kubeception.js');
 
 try {
   // inputs are defined in action metadata file
   const distribution = core.getInput('distribution');
   const version = core.getInput('version');
-  console.log(`Deleting ${distribution} ${version}!`);
   // Get the JSON webhook payload for the event that triggered the workflow
   //const payload = JSON.stringify(github.context.payload, undefined, 2)
   //console.log(`The event payload: ${payload}`);
+
+  const clusterName = process.env['clusterName']
+
+	switch(distribution) {
+	   case "Kubeception": {
+	      kubeception.deleteKluster(clusterName).then(
+	        console.log(`Deleting Kluster ${clusterName}!`)
+	      );
+	      break;
+	   }
+	   default: {
+			  console.log(`Deleting ${distribution} ${version}!`);
```

Comment:
Should the default be that the distribution isn't understood?

Reply:
You're right, but this is temporary and will change when Rafi adds his GKE code. 
Also, test matrix has included GKE and I didn't want to always show an error on those cases.